vizro-ai/src/vizro_ai/_vizro_ai.py

```diff
@@ -27,25 +26,17 @@ def __init__(self, model_name: str = "gpt-3.5-turbo-0613", temperature: int = 0)
             temperature: Temperature parameter for LLM.
 
         """
-        self.model_name = model_name
```

Comment:
FYI depending on what we interpret as constituting public API (i.e. is it public properties or just stuff that's documented), this is a breaking change. It's fine to make this change because it's not documented as a property and I'm sure no one's relying on it, but this is what an example of what I mean when I say that the structure seems a bit weird right now.

Reply:
you are right probably reverting those two properties would be better here, but letting llm to overwrite the models already makes it an interim/temp solution to just allow azure openai. We can refactor if necessary about this class